jwst/stpipe/core.py

```diff
@@ -14,6 +14,11 @@
 
 
 class JwstStep(Step):
+
+    spec = """
+    output_ext         = string(default='.fits')     # Default type of output
```

Comment:
I did some testing this morning and discovered that identically named parameters in subclasses automatically override the parent class's definition.  I think we can merge this without waiting for an stpipe release.

Reply:
Can confirm and is by design, I believe.